src/components/MyAccount/FeesBanner.tsx

```diff
@@ -1,28 +1,25 @@
-import {
-  Icon,
-  Link,
-  Notification,
-  Text,
-} from "@nypl/design-system-react-components"
-import styles from "../../../styles/components/MyAccount.module.scss"
+import { Link, Banner, Text } from "@nypl/design-system-react-components"
 import { appConfig } from "../../config/config"
 
 const FeesBanner = () => {
   return (
-    <Notification
-      notificationType="warning"
-      showIcon={false}
-      className={styles.feesNotification}
-      notificationContent={
+    <Banner
+      sx={{ "> div ": { maxWidth: "unset" }, marginBottom: "m" }}
```

Comment:
Why did you have to remove the margins?

Reply:
Design doesn't have them

Reply:
I will speak with Apoorva about this.  Those designs are not using the actual banner component, so the max-width is not built-in.  But is should be.